apps/balancer-tools/src/app/apr/(utils)/getFilteredApiUrl.ts

```diff
@@ -47,14 +51,13 @@ function getFilterDataFromParams(searchParams: SearchParams) {
   return result;
 }
 
-export default function getFilteredDateApiUrl(
+function generateQueryParams(
```

Comment:
there's some duplication in this file that made things a bit harder to parse, sth to keep an eye on